src/sentry/static/sentry/app/views/performance/landing.tsx

```diff
@@ -13,12 +14,20 @@ import Alert from 'app/components/alert';
 import EventView from 'app/utils/discover/eventView';
 import {getUtcToLocalDateObject} from 'app/utils/dates';
 import {getParams} from 'app/components/organizations/globalSelectionHeader/getParams';
-import {StyledPageHeader} from 'app/views/eventsV2/landing';
+import space from 'app/styles/space';
+import Button from 'app/components/button';
+import ButtonBar from 'app/components/buttonBar';
 
 import {generatePerformanceEventView, DEFAULT_STATS_PERIOD} from './data';
 import Table from './table';
 import Charts from './charts/index';
 
+type FilterViewType = 'ALL_TRANSACTIONS' | 'KEY_TRANSACTIONS';
+const FILTER_VIEWS: Readonly<FilterViewType[]> = [
+  'ALL_TRANSACTIONS',
+  'KEY_TRANSACTIONS',
+] as const;
```

Comment:
Could this be an enum? Seems like that would help cover both the union and lookup scenario.

Reply:
Done in https://github.com/getsentry/sentry/pull/17954/commits/90c2ea1cb9260ab39a5a6a471f710dfd222f30a8